bootstrap/environment/variables.go

```diff
@@ -44,6 +44,7 @@ const (
 	envKeyConfigDir       = "EDGEX_CONFIG_DIR"
 	envKeyProfile         = "EDGEX_PROFILE"
 	envKeyConfigFile      = "EDGEX_CONFIG_FILE"
+	envKeyRequestTimeout  = "EDGEX_SERVICE_REQUESTTIMEOUT"
```

Comment:
Now thinking this need to be unique to Uri for files
```suggestion
	envKeyRequestTimeout  = "EGDEX_FILE_URI_TIMEOUT"
```